ethers-contract/src/multicall.rs

```diff
@@ -0,0 +1,204 @@
+extern crate hex;
```

Comment:
```suggestion
```

No need for this since Rust 2018. Also, I have been using `rustc_hex` across the repo, so please use that instead of `hex`

Reply:
I have removed this. I think I also don't need the `rustc_hex` here, simply using `Address::from_str` should do the job!